ci/build_python.sh

```diff
@@ -12,14 +12,14 @@ rapids-print-env
 rapids-logger "Build rapids-xgboost"
 
```

Comment:
Though would suggest we put this up near `CONDA_CONFIG_FILE` above

```suggestion

CONDA_OVERRIDE_CUDA="${RAPIDS_CUDA_VERSION}"

```